ledger/src/check_transaction_basic.rs

```diff
@@ -24,4 +24,13 @@ impl<N: Network, C: ConsensusStorage<N>> Ledger<N, C> {
     ) -> Result<()> {
         self.vm().check_transaction(transaction, rejected_id, rng)
     }
+
+    /// Checks that the given list of transactions are well-formed and unique.
+    pub fn check_transactions_basic<R: CryptoRng + Rng>(
```

Comment:
Do you have a concrete usecase in mind for this function (in snarkOS)?

Reply:
Not particularly and we may not need this in snarkOS, but could be nice to have an interface to verify a bunch of transactions in parallel.